### ISSUE TITLE
miniview conflicts with most editors (vim/emacs/nano/...)

### ISSUE THREAD

**Issue by daveola** [borisfaure/terminology#147]
Running terminology with the miniview, if I open an editor then the editor tries to use the entire window including the columns that are obscured by miniview.

The consequence is that text on the far right columns of a file (if the width of the window or greater) cannot be read.

Perhaps there is some issue with terminology still reporting it's full size to the editor instead of the size minus the miniview columns.

**Comment by borisfaure** [comment]
This is a well know issue that I hope I will tackle some day :(

**Comment by kubijo** [comment]
Yep, the issue is not only with full-screen apps, but in general with any long text.
